### DISCUSSION TITLE
password change disable in client ui?

### DISCUSSION THREAD

**Discussion by tooptoop4** [drakkan/sftpgo#853] [General]
i am using external auth to handle logins. to avoid confusion i would like to remove 'change password' button from client web ui, how to do this?

also, is it possible to use client UI without admin UI? i am getting "The page you are looking for does not exist." if i don't setup admin password
/web/client redirects to /web/admin/setup and shows 'The page you are looking for does not exist.' and in the logs i see "message":"error getting jwt token: no token found".
my httpd config has "bindings": [{"port": 8080,"enable_web_admin": false,"hide_login_url": 3}]

cc @drakkan

**Comment by drakkan** [comment]
https://github.com/drakkan/sftpgo#support-policy

**Reply by gpointer** [reply]
Hallo Nicola
, we want to use AD , that works, but if a user change the password, system crashes.
In your answer here ,i couldn't see a solution?
Could you tell me how to disable password change for users?!

Thank you Gina
